crypto/src/hash.rs

```diff
@@ -68,6 +72,10 @@ macro_rules! define_hash {
                 }
             }
 
+            pub fn to_bytes(&self) -> &[u8] {
```

Comment:
Since we are borrowing, this should be `as_bytes` and not `to_bytes`

Reply:
will do